src/pages/services/aem-cf-editor/api/rte-toolbar/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Rich Text Editor Toolbar - AEM Content Fragments Editor Extensibility
+description: Learn what is possible to extend and customize in AEM Content Fragments Editor
+contributors:
+  - https://git.corp.adobe.com/dx-devex-acceleration/uix-docs
+---
+
+# Rich Text Editor Toolbar
+
+A Rich Text Editor is a core part of content editing software. With toolbar extensions you can add new buttons which modify the content:
+```js
+{
+  rte: {
+    getButtons: () => ([{
+      id: "test-id",
+      text: "Test button", //optional text near button
+      tooltip: "Button alt text",
+      icon: 'Airplane', //Spectrum workflow icon code from https://spectrum.adobe.com/page/icons/
+      onClick: () => {
```

Comment:
maybe for example worth some trivial usage of `state`.

Reply:
fixed